frontend/src/scenes/data-warehouse/external/DataWarehouseExternalScene.tsx

```diff
@@ -1,139 +1,81 @@
-import { IconDatabase, IconGear } from '@posthog/icons'
-import { LemonButton, Link } from '@posthog/lemon-ui'
+import { IconGear } from '@posthog/icons'
+import { LemonButton, LemonTabs, Link } from '@posthog/lemon-ui'
 import { useActions, useValues } from 'kea'
 import { router } from 'kea-router'
-import { DatabaseTableTree, TreeItem } from 'lib/components/DatabaseTableTree/DatabaseTableTree'
-import { EmptyMessage } from 'lib/components/EmptyMessage/EmptyMessage'
 import { PageHeader } from 'lib/components/PageHeader'
 import { FEATURE_FLAGS } from 'lib/constants'
-import { IconDataObject } from 'lib/lemon-ui/icons'
 import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
-import { DatabaseTable } from 'scenes/data-management/database/DatabaseTable'
 import { SceneExport } from 'scenes/sceneTypes'
 import { urls } from 'scenes/urls'
 
-import { NodeKind } from '~/queries/schema'
-
-import { DataWarehouseRowType, DataWarehouseTableType } from '../types'
+import { DataWarehouseSceneTab } from '../types'
+import { viewLinkLogic } from '../viewLinkLogic'
+import { DataWarehouseJoins } from './DataWarehouseJoins'
 import { dataWarehouseSceneLogic } from './dataWarehouseSceneLogic'
-import SourceModal from './SourceModal'
+import { DataWarehouseTables } from './DataWarehouseTables'
 
 export const scene: SceneExport = {
     component: DataWarehouseExternalScene,
     logic: dataWarehouseSceneLogic,
 }
 
+const TABS_TO_CONTENT = {
+    [DataWarehouseSceneTab.Tables]: {
+        label: 'Tables',
+        content: <DataWarehouseTables />,
+    },
+    [DataWarehouseSceneTab.Joins]: {
```

Comment:
Do we need this tab if the joins are also listed on the tables themselves?

Reply:
They're not listed in the tables UI yet. We also can't distinguish between a normal HogQL Lazy Join (e.g. `events.pdi`) vs a Data Warehouse Join (..just yet). Maybe we can fold the UI joins in eventually, but we don't have any edit functionality in the table lists at the moment and so feel like it may be a bit odd